### PR TITLE
Feat: UPHRH  Search filter added and  integration for  candidate list screen

### PR DESCRIPTION
**Pull request by sohailamjad12**
# SunbirdEd - Portal
---

### Search filter added and  integration for  candidate list screen
---

#### Example
- [x] Applicable
- [ ] Not applicable

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/client/src/app/modules/workspace/components/assessments/pending-for-submission-assessments/pending-for-submission-list.component.html [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<div class="workspace-gap">
```
Comment (devendra-tarento) [inline src/app/client/src/app/modules/workspace/components/assessments/pending-for-submission-assessments/pending-for-submission-list.component.html]: @sohailamjad12 There is already existing css classes for spacing (padding, margin). You can use here class named- mt-10. Similar way u have other class names

File: src/app/client/src/app/modules/workspace/components/assessments/pending-for-submission-assessments/pending-for-submission-list.component.scss [hunk 2]
```diff
@@ -91,3 +91,7 @@
     resize: none;
 }
 
+.workspace-gap {
```
Comment (devendra-tarento) [inline src/app/client/src/app/modules/workspace/components/assessments/pending-for-submission-assessments/pending-for-submission-list.component.scss]: @sohailamjad12 Remove this as we don't need it.

File: src/app/client/src/app/modules/workspace/components/assessments/students-list/students-list.component.html [hunk 3]
```diff
@@ -1,3 +1,7 @@
+<div class="workspace-gap">
```
Comment (devendra-tarento) [inline src/app/client/src/app/modules/workspace/components/assessments/students-list/students-list.component.html]: @sohailamjad12 Use existing classes for spacing.

File: src/app/client/src/app/modules/workspace/components/assessments/students-list/students-list.component.scss [hunk 4]
```diff
@@ -91,3 +91,7 @@
     resize: none;
 }
 
+.workspace-gap {
```
Comment (devendra-tarento) [inline src/app/client/src/app/modules/workspace/components/assessments/students-list/students-list.component.scss]: @sohailamjad12 remove this.

File: src/app/client/src/app/modules/workspace/components/assessments/pending-for-submission-assessments/pending-for-submission-list.component.scss (hunk 2 above)
Comment (sohailamjad12) [reply src/app/client/src/app/modules/workspace/components/assessments/pending-for-submission-assessments/pending-for-submission-list.component.scss]: resolved

File: src/app/client/src/app/modules/workspace/components/assessments/students-list/students-list.component.html (hunk 3 above)
Comment (sohailamjad12) [reply src/app/client/src/app/modules/workspace/components/assessments/students-list/students-list.component.html]: resolved

**Pull request merged**